### PR TITLE
Yaml documentation

### PR DESCRIPTION
**Pull request by Dhruvil2911**

### CODE REVIEW

**Comment by jdtsmith** [comment]
Thanks for working on this.  I think it's OK to merge as is and improve later.  There is a lot of info pertinent info missing from the Wiki text I wrote.  Probably good to have a high level format with a pointer to the Wiki (which we can reorganize to make it more "permanent"). I'd also suggest therefore dropping the full example, as that's in the Wiki.  I can also clean that up and check that in so we get versioning on it; it's in org not rst, but most people will read online anyway and won't know the difference I guess.

**Review by els1 — CHANGES_REQUESTED** [review]
Overall looks good. Some minor comments.

File: docs/file_formats.rst [hunk 1]
```diff
@@ -43,3 +67,192 @@ Fit Outputs
...
+ ################################
+ ionic_lines:
+     kind: line
+     wavelength:
+         '[ArII]':     6.985274
+         '[ArIII]':    8.99138
+         '[SIV]':     10.5105
+         '[NeII]':    12.813
+         '[NeIII]':   15.555
+          '[SIII]_18': 18.713
```
[the remark this answers is not in the document]
Comment (Dhruvil2911) [reply docs/file_formats.rst]: Fixed the error.

**Review by els1 — APPROVED** [review]

**Review by els1 — APPROVED** [review]

**Review by els1 — CHANGES_REQUESTED** [review]
I think your last commit included the changes of your other PR. Could you update this PR so you only commit changes related to the Yaml documentation?

**Comment by Dhruvil2911** [comment]
Yeah I've reset the PR just to the changes in the documentation.

**Review by els1 — APPROVED** [review]

**Comment by jdtsmith** [comment]
@drvdputt can you take a look at this and see if it may be useful for your documentation efforts?

**Comment by drvdputt** [comment]
I have a placeholder "YAML format description here" in my documentation (on the same page). So yes, some parts of this will be useful, even if a bit outdated. There will be a merge conflict, but since it's only documentation, a bad merge won't break anything. I can just reorganize the different parts.

**Pull request merged**